Error: Failed to load transformer

### Environment

- Operating System: Windows_NT
- Node Version:     v23.5.0
- Nuxt Version:     3.15.4
- CLI Version:      3.21.1
- Nitro Version:    2.10.4
- Package Manager:  npm@11.0.0
- Builder:          -
- User Config:      modules, ssr, app, css, site, content, runtimeConfig, experimental, compatibilityDate, nitro, vite, cname, commitShaFileGenerator, eslint, googleFonts, icon, linkChecker, sitemap
- Runtime Modules:  @nuxt/eslint@1.0.1, nuxt-cname-generator@1.0.1, ~/modules/commit-sha-file-generator, ~/modules/content-downloader, ~/modules/latex-pdf-generator, ~/modules/nuxt-c
ontent-latex, @bootstrap-vue-next/nuxt@0.26.25, @nuxtjs/google-fonts@3.2.0, nuxt-link-checker@4.1.0, @nuxtjs/sitemap@7.2.4, @nuxtjs/robots@5.2.2, @nuxt/icon@1.10.3, @nuxt/content@3.1.0
- Build Modules:    -

### Version

v3

### Reproduction

Try the sample code here : https://content.nuxt.com/docs/getting-started/configuration#transformers, on Windows.

### Description

Got the following error :

```sh
[@nuxt/content 11:35:34]  ERROR  Failed to load transformer ~/transformers/title-suffix Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'd:'

    at throwIfUnsupportedURLScheme (node:internal/modules/esm/load:205:11)
    at defaultLoad (node:internal/modules/esm/load:103:3)
    at ModuleLoader.load (node:internal/modules/esm/loader:708:12)
    at ModuleLoader.loadAndTranslate (node:internal/modules/esm/loader:488:43)
    at #createModuleJob (node:internal/modules/esm/loader:512:36)
    at #getJobFromResolveResult (node:internal/modules/esm/loader:281:34)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:249:41)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:551:25)
```

with the sample code.

<details>
<summary>nuxt.config.ts</summary>

```ts
export default defineNuxtConfig({
  content: {
    build: {
      transformers: [
        '~/transformers/title-suffix',
      ],
    },
  },
})
```

</details>

<details>

<summary>~/transformers/title-suffix.ts</summary>

```ts
import { defineTransformer } from '@nuxt/content'

export default defineTransformer({
  name: 'title-suffix',
  extensions: ['.md'],
  transform(file) {
    return {
      ...file,
      title: file.title + ' (suffix)',
    }
  },
})
```

</details>

Probably related: the base example does not work on Debian too:

```
[@nuxt/content 8:57:15 PM]  ERROR  Failed to load transformer /home/me/project/src/content/transformer Cannot find module '/home/me/project/src/content/transformer' imported from /home/me/project/node_modules/@nuxt/content/dist/module.mjs

    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1169:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:542:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:510:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:239:38)
    at ModuleLoader.import (node:internal/modules/esm/loader:472:34)
    at defaultImportModuleDynamicallyForModule (node:internal/modules/esm/utils:214:31)
    at importModuleDynamicallyCallback (node:internal/modules/esm/utils:253:12)
    at node_modules/@nuxt/content/dist/module.mjs:1395:5
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - feat/*
+      - fix/*
   pull_request:
     branches:
       - main

--- a/src/utils/content/index.ts
+++ b/src/utils/content/index.ts
@@ -6,6 +6,7 @@ import type { Nuxt } from '@nuxt/schema'
 import { resolveAlias } from '@nuxt/kit'
 import type { LanguageRegistration } from 'shiki'
 import { defu } from 'defu'
+import { createJiti } from 'jiti'
 import { createOnigurumaEngine } from 'shiki/engine/oniguruma'
 import { visit } from 'unist-util-visit'
 import type { ResolvedCollection } from '../../types/collection'
@@ -111,13 +112,15 @@ export async function createParser(collection: ResolvedCollection, nuxt?: Nuxt) 
     : undefined
 
   // Load transformers
+  const jiti = createJiti(nuxt.options.rootDir)
   const extraTransformers: ContentTransformer[] = await Promise.all(transformers.map(async (transformer) => {
     const resolved = resolveAlias(transformer, nuxt?.options?.alias)
-    return import(resolved).then(m => m.default || m).catch((e: unknown) => {
+
+    return jiti.import(resolved).then(m => (m as { default: ContentTransformer }).default || m).catch((e: unknown) => {
       logger.error(`Failed to load transformer ${transformer}`, e)
       return false
     })
-  })).then(transformers => transformers.filter(Boolean))
+  })).then(transformers => transformers.filter(Boolean)) as ContentTransformer[]
 
   const parserOptions = {
     pathMeta: pathMeta,

--- a/src/utils/content/index.ts
+++ b/src/utils/content/index.ts
@@ -112,7 +112,7 @@ export async function createParser(collection: ResolvedCollection, nuxt?: Nuxt) 
     : undefined
 
   // Load transformers
-  const jiti = createJiti(nuxt.options.rootDir)
+  const jiti = createJiti(nuxt?.options?.rootDir)
   const extraTransformers: ContentTransformer[] = await Promise.all(transformers.map(async (transformer) => {
     const resolved = resolveAlias(transformer, nuxt?.options?.alias)
 

